lib/resourceapply/apireg.go

```diff
@@ -0,0 +1,14 @@
+package resourceapply
+
+import (
+	"context"
+	"fmt"
+
+	apiregv1 "k8s.io/kube-aggregator/pkg/apis/apiregistration/v1"
+	apiregclientv1 "k8s.io/kube-aggregator/pkg/client/clientset_generated/clientset/typed/apiregistration/v1"
+)
+
+// ApplyAPIServicev1 applies the required API service to the cluster.
+func ApplyAPIServicev1(ctx context.Context, client apiregclientv1.APIServicesGetter, required *apiregv1.APIService, reconciling bool) (*apiregv1.APIService, bool, error) {
```

Comment:
Can we just drop `DeleteAPIServicev1`?  Seems like it was added in 0afb8a8658 (#438), but that it has never had any consumers.

Reply:
I'd leave it in so our `resourcebuilder` continues to support apply and delete for all the resources.

Reply:
But looking at this further, this is saying CVO will never apply this resource. Did CVO ever apply the resource such that there could ever be an orphan? If not then yeah, I suppose there's no reason for the delete.

Reply:
I've pushed ebb12713 -> 6e12b9f2 with a new commit that re-drops APIService handling.